### PR TITLE
Pin all the prod deps

### PR DESCRIPTION
**Pull request by federicomaffei**

### CODE REVIEW

File: src/services/github/reference.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ module.exports = writeOctokit => {
   const getReference = (options, next) => {
     options.ref = `heads/${options.branch}`;
 
-    writeOctokit.gitdata
+    writeOctokit.git
```
Comment (ArnoldZokas) [inline src/services/github/reference.js:24]: why is this changed?

Comment (federicomaffei) [reply src/services/github/reference.js:24]: there was a warning: `Deprecation: [@octokit/plugin-rest-endpoint-methods] "octokit.gitdata.*" methods are deprecated, use "octokit.git.*" instead`

**Review by ArnoldZokas — APPROVED** [review]
